src/packages/default/FileManager/scheme.html

```diff
@@ -57,10 +57,25 @@
             </gui-menu-entry>
           </gui-menu>
         </gui-menu-bar-entry>
-
-      </gui-menu-bar>
-    </gui-vbox-container>
-
+  </gui-menu-bar>
+  <!-- CONTEXT MENU -->
+  </gui-vbox-container>
```

Comment:
This markup is not valid. `gui-menu-bar-entry` should be a child of `gui-menu-bar`. You should move it there and use CSS to hide it.